### PR TITLE
Init

### PR DESCRIPTION
**Pull request by filipesperandio**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
+# Code Climate Sonar-PHP Engine
+
+[![CircleCI](https://circleci.com/gh/codeclimate/codeclimate-sonar-php.svg?style=svg&circle-token=72a9e9a49dc6a8653be6a69321012fe1d84abc3d)](https://circleci.com/gh/codeclimate/codeclimate-sonar-php)
+[![Maintainability](https://api.codeclimate.com/v1/badges/2bdcb2e92bbc0efb855b/maintainability)](https://codeclimate.com/github/codeclimate/codeclimate-sonar-php/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/2bdcb2e92bbc0efb855b/test_coverage)](https://codeclimate.com/github/codeclimate/codeclimate-sonar-php/test_coverage)
+
+`codeclimate-sonar-java` is a Code Climate engine that wraps [Sonarlint](http://www.sonarlint.org) in standalone mode.
```
Comment (chrishulton) [inline README.md]: this should be `codeclimate-sonar-php` right?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+```
+3. Run `codeclimate analyze`.
+
+## Custom configurations
+
+### Severity
+Ignore issues with severity below the minimum:
+```
+engines:
+  sonar-java:
```
Comment (chrishulton) [inline README.md]: same comment here: this should be `sonar-php` ?

**Review by chrishulton — APPROVED** [review]
Left a couple small comments - LGTM overall.  I'm not that familiar with sonar, so it might be worth also requesting review from someone who has been reviewing the sonar-java PRs (they might have some additional feedback).

File: README.md (hunk 1 above)
Comment (filipesperandio) [reply README.md]: Definitely!

**Pull request merged**